Source/RapyutaSimulationPlugins/Public/Robots/RRRobotStructs.h

```diff
@@ -562,12 +562,45 @@ struct RAPYUTASIMULATIONPLUGINS_API FRRRobotGeometryInfo
 };
 
 /**
- * @brief Sensor lidar info
+ * @brief Sensor info
  * @sa [URDF-Sensor](http://wiki.ros.org/urdf/XML/sensor)
  * @sa [SDF-Sensor](http://sdformat.org/spec?elem=sensor)
  */
 USTRUCT(BlueprintType)
-struct RAPYUTASIMULATIONPLUGINS_API FRRSensorLidarInfo
+struct RAPYUTASIMULATIONPLUGINS_API FRRSensorInfo
+{
+    GENERATED_BODY()
+
+    virtual ~FRRSensorInfo()
+    {
+    }
+    //! ROS Topic name to which lidar data is published to
+    UPROPERTY(EditAnywhere)
+    FString TopicName = TEXT("scan");
+
+    //! The coordinate frame in which lidar data is published under in the tf tree
+    UPROPERTY(EditAnywhere)
+    FString FrameId = TEXT("base_scan");
```

Comment:
sensor_base

Reply:
Yp, renamed to `sensor_frame` to be consistent with one defined in `URRROS2BaseSensorComponent`